pkg/option/config.go

```diff
@@ -2661,6 +2670,15 @@ func (c *DaemonConfig) calculateDynamicBPFMapSizes(totalMemory uint64, dynamicSi
 	} else {
 		log.Debugf("option %s set by user to %v", NATMapEntriesGlobalName, c.NATMapEntriesGlobal)
 	}
+	if !viper.IsSet(NeighMapEntriesGlobalName) {
```

Comment:
Could you maybe also update the unit tests to cover the newly introduced Neigh map size here:

https://github.com/cilium/cilium/blob/af9fd59b7ce0816794b9ab782e3b10f15d1df846/pkg/option/config_test.go#L265

and here:

https://github.com/cilium/cilium/blob/af9fd59b7ce0816794b9ab782e3b10f15d1df846/pkg/option/config_test.go#L472

?

Reply:
Agree, adding as follow-up as discussed.